install/kubernetes/helm/subcharts/certmanager/values.yaml

```diff
@@ -7,3 +7,8 @@ enabled: false
 hub: quay.io/jetstack
 tag: v0.5.0
 resources: {}
+
+podDisruptionBudget:
+  enabled: true
+  minAvailable: 1
+  # maxUnavailable: 0
```

Comment:
Do you see any reason why an istio component would want to expose this maxUnavailable attribute?

Reply:
I just followed general convention in https://github.com/helm/charts 
This pattern is generally used in many charts. no specific reason for this.
I changed them with `global.defaultPodDisruptionBudget` like `global.defaultResources`